src/java.base/share/classes/java/nio/channels/FileChannel.java

```diff
@@ -806,9 +809,10 @@ public abstract long transferFrom(ReadableByteChannel src,
      * #write(ByteBuffer)} method, except that bytes are written starting at
      * the given file position rather than at the channel's current position.
      * This method does not modify this channel's position.  If the given
-     * position is greater than or equal to the file's current size then the file will be
-     * grown to accommodate the new bytes; the values of any bytes between the
-     * previous end-of-file and the newly-written bytes are unspecified.  </p>
+     * position is greater than or equal to the file's current size then the
+     * file will be grown to accommodate the new bytes; the values of any bytes
+     * between the previous end-of-file and the newly-written bytes are
+     * unspecified.  </p>
```

Comment:
One thing that I missed in the initial round is that the description of parameter "position" says "within the file". That is no longer correct, it should say "The position within the file at which the transfer is to begin".